sysutils/pfSense-pkg-System_Patches/files/usr/local/pkg/patches.inc

```diff
@@ -227,4 +229,19 @@ function patch_remove_shellcmd() {
 	}
 }
 
+function patchlog($msg) {
+        syslog(LOG_WARNING, gettext("System Patches: {$msg}"));
+}
+
+function patch_descr($id) {
+	global $config;
+	$a_patches = &$config['installedpackages']['patches']['item'];
+	$descr = gettext(" (ID: {$a_patches[$id]['uniqid']}");
+	if (isset($a_patches[$id]['descr'])) {
+		$descr .= gettext(", DESCR: {$a_patches[$id]['descr']}");
```

Comment:
The only part of this string that should be run through `gettext()` is `DESCR`. The rest is formatting and variables. And that label is probably not necessary.

Reply:
gettext() fixed